corehq/apps/style/templates/style/includes/modal_report_issue.html

```diff
@@ -1,5 +1,5 @@
 {% load i18n %}
-{% if user.is_authenticated %}
+{% if request.user.is_authenticated %}
```

Comment:
the report an issue modal wasn't showing up on the edit submission page because of this. the only template that imports this one is /style/bootstrap3/base.html, and that already makes reference to request.user in some places, so seems like this was just an oversight